app/src/main/java/com/github/ianellis/shifts/location/AndroidLocationRepository.kt

```diff
@@ -27,17 +27,16 @@ class AndroidLocationRepository(
         private const val LOCATION_PERMISSION = Manifest.permission.ACCESS_COARSE_LOCATION
     }
 
-    override fun getLocationAsync(): Deferred<LatLng> {
+    override suspend fun getLocationAsync(): LatLng {
```

Comment:
Suspend function should not have "Async" suffix.

We can define async-style functions that invoke doSomethingUsefulOne and doSomethingUsefulTwo asynchronously using async coroutine builder with an explicit GlobalScope reference. We name such functions with "Async" suffix to highlight the fact that they only start asynchronous computation and one needs to use the resulting deferred value to get the result.

https://kotlinlang.org/docs/reference/coroutines/composing-suspending-functions.html#async-style-functions

Reply:
Ah, so you use the Async suffix when the method returns a Deffered object, rather than when it is a suspending function?

Reply:
Done